test/io/test_io.py

```diff
@@ -1897,3 +1897,17 @@ def test_invalidate_jwt_cache_when_secret_changes(tmp_path, defaultenv):
         # now the request should fail because the cached token is removed
         response = postgrest.session.get("/authors_only", headers=headers)
         assert response.status_code == 401
+
+
+def test_allow_configs_to_be_set_to_empty(defaultenv):
+    "configs that are explicitly set to empty (= " ") should not throw parse error"
```

Comment:
```suggestion
    "configs that are explicitly set to empty (= "") should not throw parse error"
```
I know it should be `""` here, but the style-check changes it to former. I guess I should just remove `(= " ")` part?

Reply:
> = ""

Maybe change it to `="<empty>"` to appease the styler?